File: mashup-recruit/src/main/java/kr/mashup/branding/MashupRecruitApplication.java
package kr.mashup.branding;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class MashupRecruitApplication {
    public static void main(String[] args) {
        System.setProperty("spring.config.name", "application,recruit");
        SpringApplication.run(MashupRecruitApplication.class, args);
    }
}

File: mashup-recruit/src/main/java/kr/mashup/branding/MashupRecruitApplication.kt
package kr.mashup.branding

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
class MashupRecruitApplication

fun main(args: Array<String>) {
    System.setProperty("spring.config.name", "application,recruit")
    SpringApplication.run(MashupRecruitApplication::class.java, *args)
}